src/main/resources/application.yml

```
@@ -10,11 +10,18 @@ queueconfig:
   action-field-queue: Action.Field
   outbound-exchange: adapter-outbound-exchange
   adapter-outbound-queue: RM.Field
+  receipt-queue: FieldWorkAdapter.Responses
+  receipt-routing-key: Receipt.Responses.binding
   case-event-exchange: events
   refusal-queue: FieldworkAdapter.Refusals
   refusal-routing-key: event.respondent.refusal
+
   consumers: 50
 
 healthcheck:
   frequency: 1000 #milliseconds
   filename: /tmp/fwmt-adapter-healthy
+
+caseapi:
+  host: caseapi
```

Assuming this will be overridden in k8s
```suggestion
  host: localhost
```

---

This uses caseapi for local (docker dev) testing.  localhost is used in the test yml.  K8s passes in it's own settings

---

I get the below exception when I'm running the app through my IDE: 
```

---

Would make it easier to run locally as `localhost` then change it in the config of docker-dev and k8s. I don't know if it's best to set it as local value though 🤔

---

I'll change to localhost and test again

---

application.yml should be the 'prod' settings minus anything that's sensitive ofc

---

Cool that makes sense to me 👍 We might need to mop up some of the other application.yml bits that use localhost in other repos e.g. [here](https://github.com/ONSdigital/census-rm-case-api/blob/2b9e2cb9493ae593900a2ea36a884cf888cb575d/src/main/resources/application.yml#L43)